src/scss/views/widgets/_spinner.scss

```diff
@@ -0,0 +1,79 @@
+.spinner {
+  min-width: 50px;
+  height: 20px;
+  text-align: center;
+  font-size: 10px;
+
+  & > div {
+    background-color: $color-white;
+    height: 100%;
+    width: 3px;
+    margin-right: 1px;
+    display: inline-block;
+
+    -webkit-animation: sk-stretchdelay 1.2s infinite ease-in-out;
+    animation: sk-stretchdelay 1.2s infinite ease-in-out;
+  }
+
+  &.blue > div {
+    background-color: $color-blue-active;
+  }
+
+  &.black > div {
+    background-color: $color-black;
+  }
+
+  &.large {
+    height: 100px;
+    width: 250px;
+
+    & > div {
+      width: 10px;
+    }
+  }
+
+  &.mobile-large {
+    height: 16vh;
+    width: 16vh;
+
+    & > div {
+      width: 2vh;
+      margin-right: 0.5vh;
+    }
+  }
+
+  .rect2 {
+    -webkit-animation-delay: -1.1s;
```

Comment:
would be cool if everywhere that we're double up on these values because of a vendor prefix `-webkit` etc, we use our tailor made `vendor()` mixin for sass!

mixin is defined here:
https://github.com/beatlightinc/component-catalog/blob/master/src/scss/_utilities.scss#L123

example usage:
https://github.com/beatlightinc/component-catalog/blob/master/src/scss/_utilities.scss#L132

which will automatically add all the browser vendor prefixes when it is compiled from sass -> css